### ISSUE TITLE
negative number in cn column of *call.cns

### ISSUE THREAD

**Issue by RysBen** [etal/cnvkit#831]
Hi there,

As I understand it, the `cn` column of `*call.cns` represents **each segment’s absolute integer copy number**, however, some negative number exists in cn column. 
![image](https://github.com/etal/cnvkit/assets/36749963/423fe0fc-c70f-4bae-814a-7536a6ff599e)

Can you help point out what went wrong?

Thanks

**Comment by etal** [comment]
Yes, this was a bug that was fixed in yesterday's v0.9.14 release.
